### PR TITLE
SWC-6687 / SWC-6659

### PR DESCRIPTION
**Pull request by hallieswan**
https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/15c64cc6-89ef-4296-9a6f-e2dedc0350b7

### CODE REVIEW

File: packages/synapse-react-client/src/components/AccountLevelBadge/AccountLevelBadge.tsx [hunk 1]
```diff
@@ -2,13 +2,15 @@ import { Box, Card, Link, Tooltip, Typography } from '@mui/material'
...
+import { ReactComponent as EnabledMFA } from '../../assets/icons/account-enabled-mfa.svg'
 
-export type AccountLevelBadgeType = 'certified' | 'validated'
+export type AccountLevelBadgeType = 'certified' | 'validated' | 'enabledMFA'
 type AccountLevelBadgeConfig = {
   label: string
   description: string
   tooltipText: string
   icon: React.ReactElement
+  linkHref: string
```
Comment (hallieswan) [inline packages/synapse-react-client/src/components/AccountLevelBadge/AccountLevelBadge.tsx:13]: Added so the MFA badge can link to a different docs page

File: packages/synapse-react-client/src/components/AccountLevelBadge/AccountLevelBadge.tsx [hunk 2]
```diff
@@ -42,10 +55,11 @@ export const AccountLevelBadge: React.FunctionComponent<
   return (
     <Card
       sx={{
-        maxWidth: 450,
+        maxWidth: 325,
+        minWidth: 250,
         padding: '30px 20px',
-        margin: '5px',
         display: 'flex',
+        width: '100%',
```
Comment (hallieswan) [inline packages/synapse-react-client/src/components/AccountLevelBadge/AccountLevelBadge.tsx:62]: Ensure that all badges have the same width, even in different rows

File: packages/synapse-react-client/src/components/AccountLevelBadges/AccountLevelBadges/AccountLevelBadges.stories.ts [hunk 3]
```diff
@@ -27,3 +30,15 @@ export const CertifiedAndValidated: Story = {
     userId: '273980',
   },
 }
+
+export const CertifiedValidatedAndEnabledMFAMock: Story = {
```
Comment (hallieswan) [inline packages/synapse-react-client/src/components/AccountLevelBadges/AccountLevelBadges/AccountLevelBadges.stories.ts:34]: Mock a user that has all three badges

File: packages/synapse-types/src/UserProfile.ts [hunk 4]
```diff
@@ -38,5 +38,6 @@ export type UserProfile = {
   notificationSettings?: Settings // Settings Contains a user's notification settings
   preferences?: UserPreference[]
   createdOn?: string // The date this profile was created
+  twoFactorAuthEnabled: boolean // Flag that indicates if the user has two factor authentication enabled.
```
Comment (nickgros) [inline packages/synapse-types/src/UserProfile.ts:41]: Should this be optional? I thought only ACT would see this for other users.

Comment (hallieswan) [reply packages/synapse-types/src/UserProfile.ts:41]: Good call! It can be seen by ACT and Synapse admins for other users -- I'll make it optional

**Review by nickgros — APPROVED** [review]

**Pull request merged**